Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1083,9 +1085,11 @@ public function toJson()
      */
     public function __clone()
     {
-        foreach ($this as $key => $val) {
-            $this[$key] = clone $val;
+        foreach ($this as $key => &$val) {
+            $val = clone $val;
         }
+        // unset reference to keep scope clear
+        unset($val);
```

Comment:
no, it is due to ``foreach ($this as $key => &$val) {``. the iterator does not return by reference, so you cannot assign by reference here